module/support/logging/DataLogging/data/config/webotsrobocup/DataLogging.yaml

```diff
@@ -0,0 +1,17 @@
+output:
+  directory: log
+  split_size: 5 * GiB
+
+# You can put any messages that is emitted in the system here and it will be logged
+messages:
+  message.output.CompressedImage: true
```

Comment:
As I understand we'll be getting `Image`s from webots, not `CompressedImage`s. So we probably need to setup the image compressor to do the conversion, if it isn't already.

Reply:
I think we just need to have ImageCompressor in the role and it'll do the compression when it gets an image? Then it'll log that image.

Reply:
Yep, seems so.